lib/node-token-validation/src/api/TokenValidator.ts

```diff
@@ -35,9 +36,9 @@ export class TokenValidator {
      * Constructor for the TokenValidator class
      * @param configuration Object for the TokenValidator instance
      */
-    constructor(configuration: Configuration) {
+    constructor(configuration?: Configuration) {
         // Set the configuration
-        this.config = buildConfiguration(configuration);
+        this.config = configuration ? buildConfiguration(configuration) : buildConfiguration();
```

Comment:
Is this ternary necessary? It seems the configuration object is also optional in the buildConfiguration method so we could just pass it in even if it's undefined.

Reply:
Thanks for catching it! Forgot to remove this after the configuration change. Updated.